src/Tizen.Multimedia/MediaTool/MediaPacket.cs

```diff
@@ -252,6 +252,79 @@ public bool IsEncoded
             }
         }
 
+        /// <summary>
+        /// Gets or sets the rotation value of the current packet.
+        /// </summary>
+        /// <exception cref="ArgumentException">The specified value to set is invalid.</exception>
+        /// <exception cref="ObjectDisposedException">The MediaPacket has already been disposed of.</exception>
+        /// <since_tizen> 5 </since_tizen>
```

Comment:
it needs to add InvalidOperationException for ValidateNotLocked.

Reply:
Done